### PR TITLE
Update badges in README.md

### PR DESCRIPTION
**Pull request by consideRatio**
- github test workflow badge presenting error fixed
- github issues + discourse forum badge added
- pypi badge standardized
- conda-forge badge added (may not be used, but easier to remove than to add)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,11 @@
...
+[![Latest PyPI version](https://img.shields.io/pypi/v/jupyterhub-python-repo-template?logo=pypi)](https://pypi.python.org/pypi/jupyterhub-python-repo-template)
+[![Latest conda-forge version](https://img.shields.io/conda/vn/conda-forge/jupyterhub-python-repo-template?logo=conda-forge)](https://anaconda.org/conda-forge/jupyterhub-python-repo-template)
+[![GitHub Workflow Status - Test](https://img.shields.io/github/actions/workflow/status/jupyterhub/jupyterhub-python-repo-template/test.yaml?logo=github&label=tests)](https://github.com/jupyterhub/jupyterhub-python-repo-template/actions)
+[![Test coverage of code](https://codecov.io/gh/jupyterhub/jupyterhub-python-repo-template/branch/main/graph/badge.svg)](https://codecov.io/gh/jupyterhub/jupyterhub-python-repo-template)
+[![GitHub](https://img.shields.io/badge/issue_tracking-github-blue?logo=github)](https://github.com/jupyterhub/jupyterhub-python-repo-template/issues)
+[![Discourse](https://img.shields.io/badge/help_forum-discourse-blue?logo=discourse)](https://discourse.jupyter.org/c/jupyterhub)
```
Comment (minrk) [inline README.md]: Can we use a more generic word like "forum" or "discussion"? I don't think folks should need to know what Discourse is.

**Review by minrk — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (consideRatio) [reply README.md]: The logo looks like this already:

image

```suggestion
[![Help forum - Discourse](https://img.shields.io/badge/help_forum-discourse-blue?logo=discourse)](https://discourse.jupyter.org/c/jupyterhub)
```

Comment (minrk) [reply README.md]: Matching the text in the badge is perfect!

File: README.md [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
-[![](https://img.shields.io/pypi/v/jupyterhub-python-repo-template.svg?logo=pypi)](https://pypi.python.org/pypi/jupyterhub-python-repo-template)
+[![Latest PyPI version](https://img.shields.io/pypi/v/jupyterhub-python-repo-template?logo=pypi)](https://pypi.python.org/pypi/jupyterhub-python-repo-template)
+[![Latest conda-forge version](https://img.shields.io/conda/vn/conda-forge/jupyterhub-python-repo-template?logo=conda-forge)](https://anaconda.org/conda-forge/jupyterhub-python-repo-template)
+[![GitHub Workflow Status - Test](https://img.shields.io/github/actions/workflow/status/jupyterhub/jupyterhub-python-repo-template/test.yaml?logo=github&label=tests)](https://github.com/jupyterhub/jupyterhub-python-repo-template/actions)
+[![Test coverage of code](https://codecov.io/gh/jupyterhub/jupyterhub-python-repo-template/branch/main/graph/badge.svg)](https://codecov.io/gh/jupyterhub/jupyterhub-python-repo-template)
+[![GitHub](https://img.shields.io/badge/issue_tracking-github-blue?logo=github)](https://github.com/jupyterhub/jupyterhub-python-repo-template/issues)
```
Comment (consideRatio) [inline README.md]:
```suggestion
[![Issue tracking - GitHub](https://img.shields.io/badge/issue_tracking-github-blue?logo=github)](https://github.com/jupyterhub/jupyterhub-python-repo-template/issues)
```

**Pull request merged**